prelude/main.md

```diff
@@ -17,32 +19,21 @@ pub enum TerminationInfo {
 /// `Option`/`Result`-returning function.
 macro_rules! throw {
     ($($tt:tt)*) => {
-        specr::yeet!(())
+        do yeet ()
     };
 }
 macro_rules! throw_ub {
     ($($tt:tt)*) => {
-        specr::yeet!(TerminationInfo::Ub(format!($($tt)*)))
+        do yeet TerminationInfo::Ub(format!($($tt)*))
     };
 }
 macro_rules! throw_machine_stop {
     ($($tt:tt)*) => {
-        specr::yeet!(TerminationInfo::MachineStop(format!($($tt)*)))
+        do yeet TerminationInfo::MachineStop(format!($($tt)*))
     };
 }
 
 /// We leave the encoding of the non-determinism monad opaque.
 pub use specr::Nondet;
 pub type NdResult<T=()> = specr::NdResult<T, TerminationInfo>;
-
-/// The enum `Signedness` expresses whether an integer value is signed or unsigned.
-pub use specr::Signedness;
-pub use Signedness::{Signed, Unsigned};
-
-/// Whether a pointer/reference/allocation is mutable or immutable.
-pub enum Mutability {
-    Mutable,
-    Immutable,
-}
-pub use Mutability::*;
```

Comment:
Do you want this to stay here?
I'd prefer moving this into the `libspecr::prelude`, along with `Endianness` and the `Signedness` definition.